src/Symfony/Component/Form/FormView.php

```diff
@@ -15,11 +15,17 @@
 
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @template T
+ *
+ * @implements \IteratorAggregate<string, FormView>
  */
 class FormView implements \ArrayAccess, \IteratorAggregate, \Countable
 {
     /**
      * The variables assigned to this view.
+     *
+     * @var array{value: ?T, attr: array<array-key, mixed>}&array<string, mixed>
```

Comment:
As reported in the psalm plugin repo, this is wrong. The `value` variable is not guaranteed to be of type `T` (that depends on the data transformers being applied).